src/plugins/lua/antivirus.lua

```diff
@@ -162,18 +162,24 @@ local function sophos_config(opts)
   return nil
 end
 
+local function message_not_too_large(task, rule)
+  local max_size = tonumber(rule['max_size'])
+  if not max_size then return true end
```

Comment:
I've inverted logic (so `true` means it's scanned now). :) If we don't have `max_size` setting (or it's not a number) we want to just scan.